src/components/__tests__/AeAmount.js

```diff
@@ -1,5 +1,15 @@
-import { shallowMount } from '@vue/test-utils';
+import { shallowMount, createLocalVue } from '@vue/test-utils';
+import Vuex from 'vuex';
 import AeAmount from '../AeAmount.vue';
+import Getters from '../../store/getters';
+
+const localVue = createLocalVue();
+
+localVue.use(Vuex);
+
+const store = new Vuex.Store({
+  getters: Getters,
```

Comment:
Now it looks more like an integration test than a unit test. I think the idea is to stub `roundedTokenAmount` with some simple function, test exactly AeAmount, and then test `roundedTokenAmount` in a separate file.